PathPlanning/DubinsPath/dubins_path_planning.py

```diff
@@ -255,7 +255,7 @@ def generate_local_course(total_length, lengths, mode, max_curvature,
     ll = 0.0
 
     for (m, l, i) in zip(mode, lengths, range(len(mode))):
-        if l > 0.0:
+        if l >= 0.0:
```

Comment:
when l = 0.0, we don’t need to update path points.
so it does not work?
if l = 0.0:
    continue
elseif l > 0.0:
    ….

Reply:
Makes sense to me, I broke out l == 0 like you suggested. I also tried to appease the CI errors, will change more if that isn't happy still